numba/parfor.py

```diff
@@ -410,6 +410,30 @@ def linspace_3(start, stop, num):
     ('linspace', 'numpy'): linspace_parallel_impl,
 }
 
+def fill_parallel_impl(return_type, arr, val):
```

Comment:
`return_type` is not used.

Reply:
It might be used by some other getattr function so I think we need to keep it as part of the API.